left_arm_solutions:
  -
    - [-1.089503, -0.8345749, -1.416405, -3.624608, 1.711318, 9.114411]
    - [-1.089503, -0.8345749, -1.416405, -0.4830153, -1.711318, 5.972818]
    - [-1.089503, 0.590985, 1.415262, 1.582959, 2.663702, 7.630635]
    - [-1.089503, 0.590985, 1.415262, -1.558633, -2.663702, 4.489042]
    - [2.043664, 2.878109, -0.7187063, -0.9520368, 2.549227, 8.333197]
    - [2.043664, 2.878109, -0.7187063, 2.189556, -2.549227, 5.191605]
    - [2.043664, -2.682677, 0.7175635, -0.523477, 1.998757, 8.959607]
    - [2.043664, -2.682677, 0.7175635, 2.618116, -1.998757, 5.818014]
  -
    - [-1.097679, -0.7658917, -1.352899, -3.618911, 1.709879, 9.0912]
    - [-1.097679, -0.7658917, -1.352899, -0.4773182, -1.709879, 5.949607]
    - [-1.097679, 0.5955535, 1.351756, 1.709285, 2.664317, 7.747631]
    - [-1.097679, 0.5955535, 1.351756, -1.432307, -2.664317, 4.606039]
    - [2.035713, 2.913988, -0.5890353, -0.8455978, 2.496447, 8.435895]
    - [2.035713, 2.913988, -0.5890353, 2.295995, -2.496447, 5.294302]
    - [2.035713, -2.77729, 0.5878924, -0.5267763, 2.032964, 8.915804]
    - [2.035713, -2.77729, 0.5878924, 2.614816, -2.032964, 5.774211]
  -
    - [-1.10543, -0.6960352, -1.283281, -3.613791, 1.712619, 9.065372]
    - [-1.10543, -0.6960352, -1.283281, -0.4721983, -1.712619, 5.92378]
    - [-1.10543, 0.5951586, 1.282138, 1.836365, 2.656108, 7.864992]
    - [-1.10543, 0.5951586, 1.282138, -1.305228, -2.656108, 4.723399]
    - [2.028174, 2.973733, -0.4144084, -0.7408593, 2.420826, 8.541914]
    - [2.028174, 2.973733, -0.4144084, 2.400733, -2.420826, 5.400322]
    - [2.028174, -2.893229, 0.4132655, -0.5377924, 2.087292, 8.857551]
    - [2.028174, -2.893229, 0.4132655, 2.6038, -2.087292, 5.715958]
  -
    - [-1.112789, -0.624597, -1.206853, -3.60926, 1.719849, 9.036885]
    - [-1.112789, -0.624597, -1.206853, -0.467667, -1.719849, 5.895293]
    - [-1.112789, 0.5895113, 1.20571, 1.959944, 2.638849, 7.978639]
    - [-1.112789, 0.5895113, 1.20571, -1.181648, -2.638849, 4.837046]
  -
    - [-1.119783, -0.5509897, -1.122586, -3.605375, 1.732008, 9.005609]
    - [-1.119783, -0.5509897, -1.122586, -0.4637825, -1.732008, 5.864016]
    - [-1.119783, 0.5781655, 1.121444, 2.076612, 2.612515, 8.085374]
    - [-1.119783, 0.5781655, 1.121444, -1.064981, -2.612515, 4.943781]
  -
    - [-1.126439, -0.4743504, -1.028918, -3.602263, 1.749763, 8.971281]
    - [-1.126439, -0.4743504, -1.028918, -0.4606699, -1.749763, 5.829689]
    - [-1.126439, 0.5604144, 1.027775, 2.184267, 2.577081, 8.183378]
    - [-1.126439, 0.5604144, 1.027775, -0.9573259, -2.577081, 5.041785]
  -
    - [-1.13278, -0.3933459, -0.9233575, -3.600157, 1.774182, 8.933411]
    - [-1.13278, -0.3933459, -0.9233575, -0.4585648, -1.774182, 5.791818]
    - [-1.13278, 0.5350899, 0.9222146, 2.282183, 2.532213, 8.272327]
    - [-1.13278, 0.5350899, 0.9222146, -0.8594099, -2.532213, 5.130734]
  -
    - [-1.138827, -0.3057303, -0.8016057, -3.59951, 1.80713, 8.891047]
    - [-1.138827, -0.3057303, -0.8016057, -0.4579171, -1.80713, 5.749454]
    - [-1.138827, 0.5001152, 0.8004628, 2.370843, 2.476768, 8.353346]
    - [-1.138827, 0.5001152, 0.8004628, -0.7707497, -2.476768, 5.211753]
  -
    - [-1.1446, -0.2071222, -0.655114, 2.681902, 1.852342, 8.842146]
    - [-1.1446, -0.2071222, -0.655114, -0.4596903, -1.852342, 5.700553]
    - [-1.1446, 0.451278, 0.6539711, 2.451878, 2.40766, 8.429281]
    - [-1.1446, 0.451278, 0.6539711, -0.689715, -2.40766, 5.287688]
  -
    - [-1.150116, -0.08613076, -0.4613908, 2.674985, 1.919678, 8.780998]
    - [-1.150116, -0.08613076, -0.4613908, -0.466608, -1.919678, 5.639406]
    - [-1.150116, 0.3773542, 0.4602479, 2.528997, 2.315572, 8.506963]
    - [-1.150116, 0.3773542, 0.4602479, -0.6125959, -2.315572, 5.36537]
best_left_arm_solutions:
  - [-1.089503, -0.8345749, -1.416405, -0.4830153, -1.711318, 5.972818]
  - [-1.097679, -0.7658917, -1.352899, -0.4773182, -1.709879, 5.949607]
  - [-1.10543, -0.6960352, -1.283281, -0.4721983, -1.712619, 5.92378]
  - [-1.112789, -0.624597, -1.206853, -0.467667, -1.719849, 5.895293]
  - [-1.119783, -0.5509897, -1.122586, -0.4637825, -1.732008, 5.864016]
  - [-1.126439, -0.4743504, -1.028918, -0.4606699, -1.749763, 5.829689]
  - [-1.13278, -0.3933459, -0.9233575, -0.4585648, -1.774182, 5.791818]
  - [-1.138827, -0.3057303, -0.8016057, -0.4579171, -1.80713, 5.749454]
  - [-1.1446, -0.2071222, -0.655114, -0.4596903, -1.852342, 5.700553]
  - [-1.150116, -0.08613076, -0.4613908, -0.466608, -1.919678, 5.639406]
right_arm_solutions:
  -
    - [0.7102203, -0.7757053, -1.719978, -2.695836, 1.272495, -2.721846]
    - [0.7102203, -0.7757053, -1.719978, 0.4457562, -1.272495, -5.863439]
    - [0.7102203, 0.9568929, 1.718836, -1.971907, 2.67748, -1.454382]
    - [0.7102203, 0.9568929, 1.718836, -5.1135, -2.67748, -4.595975]
    - [-2.440986, 2.473197, -1.185213, -5.462285, 2.535025, -1.85084]
    - [-2.440986, 2.473197, -1.185213, -2.320692, -2.535025, -4.992434]
    - [-2.440986, -2.6177, 1.18407, 0.4307325, 1.527249, -2.593691]
    - [-2.440986, -2.6177, 1.18407, -2.71086, -1.527249, -5.735284]
  -
    - [0.732403, -0.7127288, -1.659078, -2.70996, 1.276129, -2.703328]
    - [0.732403, -0.7127288, -1.659078, 0.4316328, -1.276129, -5.844921]
    - [0.732403, 0.9581841, 1.657935, -2.081342, 2.66491, -1.561987]
    - [0.732403, 0.9581841, 1.657935, -5.222935, -2.66491, -4.70358]
    - [-2.418546, 2.488716, -1.10287, -5.537106, 2.501669, -1.924056]
    - [-2.418546, 2.488716, -1.10287, -2.395513, -2.501669, -5.065649]
    - [-2.418546, -2.685186, 1.101727, 0.4174361, 1.548123, -2.572098]
    - [-2.418546, -2.685186, 1.101727, -2.724156, -1.548123, -5.713691]
  -
    - [0.7534156, -0.6496733, -1.593618, -2.723932, 1.283387, -2.682265]
    - [0.7534156, -0.6496733, -1.593618, 0.4176607, -1.283387, -5.823858]
    - [0.7534156, 0.954992, 1.592475, -2.184697, 2.645632, -1.661856]
    - [0.7534156, 0.954992, 1.592475, -5.32629, -2.645632, -4.803449]
    - [-2.397285, 2.510468, -1.010878, -5.606633, 2.461125, -1.99115]
    - [-2.397285, 2.510468, -1.010878, -2.46504, -2.461125, -5.132743]
    - [-2.397285, -2.756126, 1.009735, 0.4048953, 1.574259, -2.547644]
    - [-2.397285, -2.756126, 1.009735, -2.736697, -1.574259, -5.689237]
  -
    - [0.773331, -0.5862894, -1.523355, -2.737701, 1.294378, -2.658875]
    - [0.773331, -0.5862894, -1.523355, 0.4038913, -1.294378, -5.800468]
    - [0.773331, 0.9473247, 1.522212, -2.280023, 2.620049, -1.752224]
    - [0.773331, 0.9473247, 1.522212, -5.421616, -2.620049, -4.893816]
    - [-2.37713, 2.539666, -0.9067188, -5.670724, 2.412566, -2.052379]
    - [-2.37713, 2.539666, -0.9067188, -2.529131, -2.412566, -5.193973]
    - [-2.37713, -2.831839, 0.9055759, 0.3932631, 1.606886, -2.52008]
    - [-2.37713, -2.831839, 0.9055759, -2.74833, -1.606886, -5.661673]
  -
    - [0.7922181, -0.5222558, -1.447916, -2.75121, 1.309253, -2.633347]
    - [0.7922181, -0.5222558, -1.447916, 0.3903828, -1.309253, -5.77494]
    - [0.7922181, 0.9351292, 1.446773, -2.366382, 2.58859, -1.832337]
    - [0.7922181, 0.9351292, 1.446773, -5.507975, -2.58859, -4.973929]
    - [-2.358013, 2.578455, -0.7860027, -5.729688, 2.354288, -2.108658]
    - [-2.358013, 2.578455, -0.7860027, -2.588095, -2.354288, -5.250251]
    - [-2.358013, -2.914592, 0.7848598, 0.3828063, 1.648087, -2.488817]
    - [-2.358013, -2.914592, 0.7848598, -2.758786, -1.648087, -5.63041]
  -
    - [0.810142, -0.4571614, -1.36675, -2.764395, 1.328231, -2.605829]
    - [0.810142, -0.4571614, -1.36675, 0.3771975, -1.328231, -5.747421]
    - [0.810142, 0.9182643, 1.365607, -2.443659, 2.5516, -1.902272]
    - [0.810142, 0.9182643, 1.365607, -5.585252, -2.5516, -5.043864]
    - [-2.339868, 2.631208, -0.6399038, -5.784348, 2.28249, -2.161921]
    - [-2.339868, 2.631208, -0.6399038, -2.642756, -2.28249, -5.303514]
    - [-2.339868, -3.008883, 0.6387609, 0.3740797, 1.702003, -2.452451]
    - [-2.339868, -3.008883, 0.6387609, -2.767513, -1.702003, -5.594044]
  -
    - [0.8271639, -0.3904727, -1.279058, -2.77719, 1.351642, -2.57642]
    - [0.8271639, -0.3904727, -1.279058, 0.3644028, -1.351642, -5.718012]
    - [0.8271639, 0.8964608, 1.277915, -2.512298, 2.509254, -1.962677]
    - [0.8271639, 0.8964608, 1.277915, -5.653891, -2.509254, -5.10427]
    - [-2.322633, 2.709813, -0.4446479, -5.836694, 2.186389, -2.217021]
    - [-2.322633, 2.709813, -0.4446479, -2.695101, -2.186389, -5.358613]
    - [-2.322633, -3.12673, 0.4435051, 0.3686695, 1.779717, -2.406771]
    - [-2.322633, -3.12673, 0.4435051, -2.772923, -1.779717, -5.548364]
  -
    - [0.8433407, -0.3214729, -1.183666, -2.789519, 1.37999, -2.545149]
    - [0.8433407, -0.3214729, -1.183666, 0.3520737, -1.37999, -5.686742]
    - [0.8433407, 0.8692555, 1.182523, -2.573064, 2.461475, -2.014551]
    - [0.8433407, 0.8692555, 1.182523, -5.714657, -2.461475, -5.156144]
  -
    - [0.8587261, -0.2491502, -1.078801, -2.801293, 1.41406, -2.511948]
    - [0.8587261, -0.2491502, -1.078801, 0.3402994, -1.41406, -5.653541]
    - [0.8587261, 0.8358771, 1.077658, -2.626871, 2.407817, -2.059086]
    - [0.8587261, 0.8358771, 1.077658, -5.768464, -2.407817, -5.200679]
  -
    - [0.8733698, -0.1719674, -0.9616292, -2.812393, 1.455145, -2.476584]
    - [0.8733698, -0.1719674, -0.9616292, 0.3291993, -1.455145, -5.618176]
    - [0.8733698, 0.7950138, 0.9604863, -2.674681, 2.347247, -2.097624]
    - [0.8733698, 0.7950138, 0.9604863, -5.816274, -2.347247, -5.239217]
best_right_arm_solutions:
  - [0.7102203, -0.7757053, -1.719978, -2.695836, 1.272495, -2.721846]
  - [0.732403, -0.7127288, -1.659078, -2.70996, 1.276129, -2.703328]
  - [0.7534156, -0.6496733, -1.593618, -2.723932, 1.283387, -2.682265]
  - [0.773331, -0.5862894, -1.523355, -2.737701, 1.294378, -2.658875]
  - [0.7922181, -0.5222558, -1.447916, -2.75121, 1.309253, -2.633347]
  - [0.810142, -0.4571614, -1.36675, -2.764395, 1.328231, -2.605829]
  - [0.8271639, -0.3904727, -1.279058, -2.77719, 1.351642, -2.57642]
  - [0.8433407, -0.3214729, -1.183666, -2.789519, 1.37999, -2.545149]
  - [0.8587261, -0.2491502, -1.078801, -2.801293, 1.41406, -2.511948]
  - [0.8733698, -0.1719674, -0.9616292, -2.812393, 1.455145, -2.476584]